src/main/java/bio/terra/cli/command/Main.java

```diff
@@ -39,7 +39,7 @@
       Nextflow.class
     },
     description = "Terra CLI")
-class Main implements Runnable {
+public class Main implements Runnable {
```

Comment:
Just for documentation, you mean? I don't think this annotation will remove the need to make the method public, because the tests are not (and generally cannot be) in the same package as all the commands they call.

Reply:
Added annotation.